requirements/dev-requirements.txt

```diff
@@ -0,0 +1,94 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file requirements/dev-requirements.txt requirements/dev-requirements.in
+#
```

Comment:
The compiled dev-requirements.txt will be different dependent on python version. If we only plan on supporting py3.6 for galaxy going forward that should be fine. If we need to support both, may need a way to distinguish py2/py3 requirements.

Reply:
We will not support python 2 because pulp supports only python 3.